compiler/compiler.go

```diff
@@ -224,3 +226,43 @@ func (m *meter) Progress() zbuf.Progress {
 	}
 	return out
 }
+
+func ParseRangeExpr(zctx *zed.Context, src string, layout order.Layout) (*zed.Value, string, error) {
```

Comment:
I'm not really a fan of putting this in compiler, considering moving this into branch...

Reply:
This also could be gotten rid of and I could just use strings.Split instead 🤔